src/system/api/rqidApi.ts

```diff
@@ -504,28 +544,8 @@ export async function getActorTemplates(): Promise<RqgActor[] | undefined> {
 }
 
 export async function getHomelands(): Promise<RqgItem[] | undefined> {
```

Comment:
@wake42 I feel like this is probably a good start for a generic method of getting all items of any given type, but I guess so is fromRqidRegex.  I think you said I should look into that and I did this before I did the work adding the "best" to fromRqidRegex and now, upon reflection, I think I understood what you meant. I may well remove this.

Reply:
I think so, and since this is an api about rqid I don't think getHomelands belongs in this file. So maybe one could use fromRqidRegex instead - you could have a getHomelands function in the homelands item maybe if that would help.

Reply:
Getting rid of getActorTemplates and getHomelands

Reply:
Gone